server-client/index.js

```diff
@@ -35,6 +35,7 @@ export class ServerClient {
       auth: this.auth.bind(this),
       inFilter: this.filter.bind(this),
       inMap: this.inMap.bind(this),
+      onSync: app.onSync?.bind(app),
```

Comment:
```suggestion
      onSync: app.onSync.bind(app),
```

Seems like it always exists